playground/css/index.html

```diff
@@ -202,5 +202,10 @@ <h1>CSS</h1>
 <style>
   @import url(./imported.scss);
 </style>
+<style>
+  body {
+    background: red;
+  }
+</style>
```

Comment:
Wouldn't this be enough to trigger the issue? Or do we need `input: { main: 'index.html' }` to make the build fail in windows? Probably we need a new playground in that case although I don't know if we should add it only for testing this. I'll let other folks that have a window machine get involved here.

Reply:
Yes, I tested locally that this issue will be triggered only if two conditions exist at the same time. Looking forward to your test results.